tests/unit/helper.py

```diff
@@ -79,12 +79,8 @@ def create_instance_of_described_class(self):
         If cls.example_data is None, just create a simple instance of the
         class.
         """
-        if self.example_data:
-            instance = self.described_class(self.example_data,
-                                            self.session)
-        else:
-            instance = self.described_class()
-            instance.session = self.session
+        instance = self.described_class(self.example_data)
```

Comment:
At first, I had to override this method in TestGitHubEnterprise to have it behave like this. After a discussion with @ueg1990 , I realised the first part of the if I deleted above was not used anywhere, so I rewrote a simpler version which I did not need to override in the child class.